src/core/ext/filters/fault_injection/fault_injection_filter.h

```diff
@@ -45,7 +45,9 @@ class FaultInjectionFilter
  public:
   static const grpc_channel_filter kFilter;
 
-  static absl::StatusOr<FaultInjectionFilter> Create(
+  explicit FaultInjectionFilter(ChannelFilter::Args filter_args);
+
+  static absl::StatusOr<std::unique_ptr<FaultInjectionFilter>> Create(
```

Comment:
Please declare factory functions before ctors, so that it's clear to readers that that's the expected way to instantiate.

Same throughout.

Reply:
done